draft-ietf-ppm-dap.md

```diff
@@ -1845,9 +1846,9 @@ This message consists of:
   initializing an aggregation job, the Leader MUST validate the parameter as
   described in {{agg-param-validation}}.
 
-* `part_batch_selector`: The "partial batch selector" used by the Aggregators to
-  determine how to aggregate each report. Its contents depends on the indicated
-  batch mode. This field is called the "partial" batch selector because
+* `part_batch_selector`: The "partial batch selector" used by the Aggregators
+  to determine how to aggregate each report. Its contents depends on the batch
+  mode of the task. This field is called the "partial" batch selector because
```

Comment:
We put the batch mode in `PartialBatchSelector` precisely so that implementations wouldn't need to go load a task from storage in order to parse `AggregationJobInitReq`, so _actually_ (🤓) I think it's more correct to point the reader to `part_batch_selector.batch_mode` than to the task. Perhaps there's a better way to do that.

"Its contents depend on the batch mode indicated by `part_batch_selector.batch_mode`."?

note also we should remove the "s" at the end of "depends" in this sentence.

Reply:
I'll just drop it.